<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HexapodControlWidget</class>
 <widget class="QWidget" name="HexapodControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Global Functions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="0">
       <widget class="QPushButton" name="disableButton">
        <property name="text">
         <string>Disable</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="calibrateButton">
        <property name="text">
         <string>Calibrate</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="enableButton">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="allocateJointsButton">
        <property name="text">
         <string>Allocate Joints</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Leg Forces</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QLCDNumber" name="leg0ForceDisplay">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Leg 0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="leg1ForceDisplay">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Leg 1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Leg 2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Leg 3</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Leg 4</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Leg 5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLCDNumber" name="leg2ForceDisplay">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLCDNumber" name="leg3ForceDisplay">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLCDNumber" name="leg4ForceDisplay">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLCDNumber" name="leg5ForceDisplay">
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Legs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="LegPushButton" name="leg0Button">
        <property name="text">
         <string>Leg 0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LegPushButton" name="leg1Button">
        <property name="text">
         <string>Leg 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="LegPushButton" name="leg5Button">
        <property name="text">
         <string>Leg 5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="LegPushButton" name="leg3Button">
        <property name="text">
         <string>Leg 3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LegPushButton" name="leg4Button">
        <property name="text">
         <string>Leg 4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="LegPushButton" name="leg2Button">
        <property name="text">
         <string>Leg 2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="SpinSlider" name="globalXInput" native="true"/>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="stepButton">
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="SpinSlider" name="directionAngleInput" native="true"/>
      </item>
      <item row="0" column="0">
       <widget class="SpinSlider" name="heightInput" native="true"/>
      </item>
      <item row="5" column="0">
       <widget class="SpinSlider" name="globalYInput" native="true"/>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="northButton">
        <property name="text">
         <string>North</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="southButton">
        <property name="text">
         <string>South</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="eastButton">
        <property name="text">
         <string>East</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QPushButton" name="westButton">
        <property name="text">
         <string>West</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="SpinSlider" name="globalZInput" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LegPushButton</class>
   <extends>QPushButton</extends>
   <header>legpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>SpinSlider</class>
   <extends>QWidget</extends>
   <header>spinslider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
